src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -132,4 +181,32 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String hw} into a {@code Homework}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code Homework} is invalid.
+     */
+    public static Homework parseHw(String hw) throws ParseException {
```

Comment:
I'm not sure if a TimeRange class is very necessary. What do you think about having the comparison method in the Time class instead? ParserUtil actually has a constant String declared so we could do the same by adding MESSAGE_INVALID_TIME_RANGE. What are your thoughts about this?

Reply:
I think you add the constant and method into `Time` class, and then call it from `LessonAddCommandParser` like the current one

Reply:
The method I have in mind for comparing the Time objects is a general comparison, similar to compareTo() so it might not have an idea of "invalid range" and I do the logic of this in ParserUtil.parseTimeRange(...) instead, so would it be better to add the constant in ParserUtil instead?

Reply:
hmm ok I think it works too

Reply:
I think you can also make Time a `comparable`, add a `compareTo` in `Time`, before calling it in `ParserUtil`. Just a matter of design I guess